### ISSUE TITLE
Error installing Spyder 5.2.0

### ISSUE THREAD

**Issue by FcoJavFernandez** [spyder-ide/spyder#16917]
Hi...
The installation script has obvious errors.
The issue report is empty and Spyder crashes... 
This is it.

- Windows 10 actualizado
- Python 3.10 Works fine!

The issue report is empty and Spyder crashes... 
This is it.
Thanks.

![imagen](https://user-images.githubusercontent.com/77489889/144057080-b13e64eb-8de1-4b99-8bcc-14bee273a926.png)
![imagen](https://user-images.githubusercontent.com/77489889/144057101-01200b41-5726-467a-85b9-9ffb0f5b1a73.png)
![imagen](https://user-images.githubusercontent.com/77489889/144057118-19115679-82d3-4037-a85a-4e69fdef06d6.png)
![imagen](https://user-images.githubusercontent.com/77489889/144057140-ef0595f5-7d9f-4063-af94-4cc1fa00e247.png)
![imagen](https://user-images.githubusercontent.com/77489889/144057162-109d1724-be72-45f9-b9fb-faf4a4d1e678.png)
![imagen](https://user-images.githubusercontent.com/77489889/144057183-29020f6a-6b09-41c9-b604-a319982a98b8.png)
![imagen](https://user-images.githubusercontent.com/77489889/144057200-6fca3b84-d172-4f36-a77d-39af9bf9657e.png)
![imagen](https://user-images.githubusercontent.com/77489889/144057573-55a6a7dc-1136-4342-b3d5-56c320f47732.png)

**Comment by dalthviz** [comment]
Hi @FcoJavFernandez thank you for the feedback. Seems like you are trying to install Spyder in your `D` disk, right? I tried with the [installer](https://github.com/spyder-ide/spyder/releases/download/v5.2.0/Spyder_64bit_full.exe) in an external disk (which ends up mapped as `D` for me) but it worked.

Also, seems like you have Python 3.10 installed? How did you install that Python version?

Any new info to replicate the issue is greatly appreciated. Let us know!

**Comment by FcoJavFernandez** [comment]
Hi.

> I guess you won't believe you're talking to a bibiclete mechanic, right?

In my PC **D**: is an internal hard disk, and **C**: is an internal SSD disk.
I have installed **Python 3.10** from the Python website and it works perfectly, even in the previous version of Spyder (**I have installed and uninstalled Spyder several times...**) or PyCharm...

![imagen](https://user-images.githubusercontent.com/77489889/144101366-3806df31-37ec-4307-b012-94c17749fb32.png)

![imagen](https://user-images.githubusercontent.com/77489889/144101705-02928e5e-74f3-4097-9203-c2cf365727d2.png)


![imagen](https://user-images.githubusercontent.com/77489889/144098508-22e6234a-a5c4-450b-b202-2659a9ee9606.png)

**`By the way, in the images you can see the "**errors**" in the installation script...`**

![imagen](https://user-images.githubusercontent.com/77489889/144100661-3ae08fb3-13db-4937-bdf3-8b3dcc2f5847.png)


Thanks.

**Comment by Hillard28** [comment]
Have you tried reinstalling spyder-kernels v2.2.0?  That seems to have fixed it for me, as I was having repeated crashes on my C: drive after updating (I uninstalled previous installation then installed v5.2.0).

conda install -c conda-forge spyder-kernels=2.2.0

**Comment by dalthviz** [comment]
@FcoJavFernandez do you use a custom interpreter with the installer?

**Comment by FcoJavFernandez** [comment]
No, I do the installation with the official exe, and also with the reduced installer, and I don't have Anaconda installed.

I have reinstalled the kernel 2.2.0 with pip and it has not taken effect, neither reinstall Spyder with pip...
I just uninstalled Spyder.

**Comment by FcoJavFernandez** [comment]
By doing tests with Python, I just generated an exe with pyinstaller and it has generated it perfectly...

![imagen](https://user-images.githubusercontent.com/77489889/144124454-3fb46226-b51a-4b5a-befa-943750923bae.png)


![imagen](https://user-images.githubusercontent.com/77489889/144124217-f490f825-ee98-4e66-8ceb-d1752f00bdda.png)

**Comment by FcoJavFernandez** [comment]
This is the error that occurs at the end of the process with pip...

![imagen](https://user-images.githubusercontent.com/77489889/144126391-781a4a9e-ad35-468c-88ab-5ed399fd4d39.png)

**Comment by ccordoba12** [comment]
@FcoJavFernandez, please remove the directory `C:\Users\corta\.spyder-py3`. That could help to solve the problem your seeing with our installer.

In the last comment you said:

> This is the error that occurs at the end of the process with pip...

I'm afraid that means Spyder can't be installed on Windows with pip when using Python 3.10. That's because one of our dependencies (`psutil`) doesn't offer binary packages for that version yet. So you need to downgrade to Python 3.9 instead, sorry.

**Comment by FcoJavFernandez** [comment]
Hello, @ccordoba12, when deleting the folder **.spyder-py3**, Spyder starts well, but it still gives some problems and you have to delete it again to solve them...
At least now you can see the information window.

> In this case there were two configuration windows and the first one tried to restart Spyder to apply the changes, but the second one gave the error...

![imagen](https://user-images.githubusercontent.com/77489889/144205163-3693d234-5b3b-49ec-82fa-5ac98f694b0b.png)

To resolve this last error it was not necessary to delete the folder. spyder, but it's one problem after another...

> **Anyway, since Spyder fails if I try to use Python 3.10 and I have to install in Python 3.7.9 the missing modules like Pygame, etc., it is not worth continuing to test.**

Thanks a lot.

**Comment by ccordoba12** [comment]
> At least now you can see the information window

Ok, that's good. Please copy the contents of the error you're getting in the report window and post them here to see what's happening in your case.

**Comment by FcoJavFernandez** [comment]
That would be very good, and I would gladly do it, if I had time to do something with the happy screen before it turns off by itself ...
Hence the screenshot, which is the only thing that gives time, sometimes.

**Comment by ccordoba12** [comment]
Ok, then that could be caused by the bug reported on issue #16896, which will be fixed in our next version (5.2.1), to be released shortly.
